app/components/modals/cropper-modal.js

```diff
@@ -4,9 +4,8 @@ export default class extends ModalBase {
   onVisible() {
     let viewport = {};
     let factor = 150;
-    const aspectRatio = this.getWithDefault('aspectRatio', [2, 1]);
```

Comment:
This will break the default value of the prop

Reply:
@iamareebjamal i have added it in image-upload.js , so default value always be [2, 1]

Reply:
We don't want that